src/components/Order.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { OrderItem } from "./OrderItem";
+
+export class OrderSummary extends React.Component {
+  render() {
+    const { items, total, removeFromOrder } = this.props;
+
+    return (
+      <div className="order">
+        <h1>Order</h1>
+        <ul className="order__items">
+          {items.map((item, i) => {
+            const { name, price, bread, isHot } = item;
+            return (
+              <OrderItem
+                key={i}
+                index={i}
```

Comment:
I don't think this is needed as its the same value as the `key`, in `OrderItem` component you can use the `key` value instead of `index`